terraform/modules/task-definitions/frontend/main.tf

```diff
@@ -27,8 +27,8 @@ module "task_definition" {
   source             = "../../task-definition"
   mesh_name          = var.mesh_name
   service_name       = var.service_name
-  cpu                = 512
-  memory             = 1024
+  cpu                = 1024
+  memory             = 2048
```

Comment:
Was it necessary to increase CPU/memory?

Reply:
It was close to the limit on startup. Maybe we should look into that in more detail later.